front/app/components/ScreenReadableEventDate/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import moment from 'moment';
+
+import { IEventData } from 'api/events/types';
+
+import { ScreenReaderOnly } from 'utils/a11y';
+import { useIntl } from 'utils/cl-intl';
+
+import messages from './messages';
+
+interface Props {
+  event: IEventData;
+}
+
+const ScreenReadableEventDate = ({ event }: Props) => {
```

Comment:
Could you add a word or two on why this component is needed rather than the date bits that already were in place?